treestats_paper.tex

```diff
@@ -645,15 +660,17 @@ \subsection*{Branch statistics}
     \caption{
     Computing the $f_4(A,B;C,D)$ statistic on a tree.
     \textbf{(a)} Each mutation is given weight equal to $(p_A - p_B)(p_C - p_D)$,
-    where $p_X$ is the proportion of the samples inheriting that mutation in set $X$
+    where $p_X$ is the proportion of the samples inheriting that mutation (colored stars) in set $X$
     (shown in labels at the bottom).
     Note that each of the mutations shown occurs at a distinct site, so there is no backmutation.
     \textbf{(b)} Each branch is assigned weight equal to the weight that would be assigned
-    to any mutation falling in it, and then the branch $f_4$ statistic
+    to any mutation falling in it (colored lined), and then the branch $f_4$ statistic
     is the sum over these weights, multiplied by the length of the branch
     (and the span of the tree, not depicted).
         \label{fig:branch_site_diagram}
     }
+    \krt{This figure is not as helpful as fig 1. It takes a lot of fiddling to work out the values in each panel. Is it
+    possible to fill it out as in fig 1, or is that just too much space?}
```

Comment:
Hm. Not sure. The main point of this figure is conceptual, to make the point that each branch gets the value per unit length of whatever a mutation on that branch would contribute. There isn't even enough info in this figure to get the values totally (you need branch lengths too), and adding all that would make it a lot more cluttered. What if we did it so Figures 1 & 2 talked about the same statistic, so the values came from Figure 1? I'd say that if so, we shouldn't use f4 so we don't have to have a list of four numbers at each node, though. What do you think?

Reply:
This sounds like a good idea.